Sources/Testing/Issues/Issue.swift

```diff
@@ -49,12 +49,10 @@ public struct Issue: Sendable {
     ///
     /// - Parameters:
     ///   - timeLimitComponents: The time limit reached by the test.
-    ///
-    /// @Comment {
-    ///   - Bug: The associated value of this enumeration case should be an
-    ///     instance of `Duration`, but the testing library's deployment target
-    ///     predates the introduction of that type.
-    /// }
+    //
+    // - Bug: The associated value of this enumeration case should be an
```

Comment:
I confirmed that the rendered DocC still shows the documentation block comments above, despite having 2-slash `//` comments in between.